docs/reference/migration/migrate_7_0/api.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[breaking_70_api_changes]]
+=== Breaking changes in 7.0
+
+==== Deprecated camel case and underscore parameters
```

Comment:
The deprecation is for 6.x so here you should say something like:
Camel case and underscore parameters deprecated in 6 have been removed.